### DISCUSSION TITLE
Suggest documentation warning of error message "Cannot infer contextual base in reference to member ______"

### DISCUSSION THREAD

**Discussion by danwood** [pointfreeco/swift-html#84] [General]
I've been getting this error message as I build up HTML code, and I finally realized that what was wrong was a missing comma, for instance after the `.header(…)` here. It might be good to document the error message phrase  _"Cannot infer contextual base in reference to member ______"_ for people who encounter this error in the future.

<img width="985" alt="Screen Shot 2020-06-30 at 8 44 58 PM" src="https://user-images.githubusercontent.com/207080/86200863-bc3ad880-bb12-11ea-99ac-fb903fd357c5.png">

(This is using Xcode 12 beta; don't know how it will appear in older or newer versions of the Swift compiler…)

**Comment by mbrandonw** [comment]
Hey @danwood, this is a good suggestion! We'll look into documenting it soon. Thanks!

**Comment by stephencelis** [comment]
I think this has improved in later versions of Xcode. I'm going to convert this to a discussion, in case there are other ideas around documenting some of Swift's gnarlier diagnostics.
